QuantExt/qle/instruments/crossccybasisswap.hpp

```diff
@@ -48,9 +48,9 @@ class CrossCcyBasisSwap : public CrossCcySwap {
         holds the receive currency cashflows.
     */
     CrossCcyBasisSwap(Real payNominal, const Currency& payCurrency, const Schedule& paySchedule,
-                      const boost::shared_ptr<IborIndex>& payIndex, Spread paySpread, Real recNominal,
+                      const boost::shared_ptr<IborIndex>& payIndex, Spread paySpread, Real payGearing, Real recNominal,
```

Comment:
Added the parameters payGearing and recGearing right after the existing paySpread and recSpread respectively.
In order to support these two new parameters, I had to introduce the corresponding private elements payGearing_ and recGearing_.